lib/ansible/modules/network/nxos/nxos_vrf.py

```diff
@@ -224,12 +224,12 @@ def map_obj_to_commands(updates, module):
     state = module.params['state']
     purge = module.params['purge']
 
+    args = ('rd', 'description', 'vni')
+
     for w in want:
         name = w['name']
-        description = w['description']
-        vni = w['vni']
-        rd = w['rd']
         admin_state = w['admin_state']
+        vni = w['vni']
```

Comment:
This is not required also like description and rd. Anyway you are setting it below from want

Reply:
This is required for L256